Algebroids/tst/AdditiveClosureOfAlgebroid_vs_QuiverRows.tst

```diff
@@ -43,7 +43,7 @@ gap> runtime_quiver := Runtime( ) - start;;
 #
 gap> if runtime >= runtime_quiver * 3 / 10 then Display( true ); else Display( runtime ); Display( runtime_quiver ); fi;
 true
-gap> if runtime <= runtime_quiver * 4 / 10 then Display( true ); else Display( runtime ); Display( runtime_quiver ); fi;
+gap> if runtime <= runtime_quiver * 5 / 10 then Display( true ); else Display( runtime ); Display( runtime_quiver ); fi;
```

Comment:
If this is needed, this means that the compiled code has gotten slower compared to the non-compiled code. Which commit is responsible for this?

Reply:
I can only reproduce this on my laptop, where the noncompiled code seems to be faster than in the CI.

Reply:
Has this always been the case, i.e. already when this test was introduced?

Reply:
No, this is new.

Reply:
I will wait for your answer before I merge.

Reply:
I have a long queue of commits, I have to commit in 20 minutes and we can continue discussing this point later.

Reply:
> No, this is new.

Could you find out since which commit the behaviour has changed? We really do not want to have unexpected regressions in the system, which is why we have this rather strict test in the first place.

Reply:
> Could you find out since which commit the behaviour has changed? We really do not want to have unexpected regressions in the system, which is why we have this rather strict test in the first place.

Yes, it is on my todo list for later tonight :)

Reply:
Awkward enough, the regression is caused by PR https://github.com/homalg-project/HigherHomologicalAlgebra/pull/158:

Being on https://github.com/homalg-project/CategoricalTowers/pull/271

* The regerssion does not occur with PR https://github.com/homalg-project/HigherHomologicalAlgebra/pull/156
* The regerssion occurs  PR https://github.com/homalg-project/HigherHomologicalAlgebra/pull/158

Reply:
Can you post `runtime` and `runtime_quiver` before and after the regression?

Reply:
Without regression:

```gap
AdditiveClosureOfAlgebroid_vs_QuiverRows
msecs: 3367
gap> runtime;
666
gap> runtime_quiver;
1586
gap> Float( runtime_quiver * 4 / 10 );
658.8
gap> Float( runtime_quiver * 4 / 10 );
634.4
```

---
With regression:

```gap
AdditiveClosureOfAlgebroid_vs_QuiverRows
msecs: 3309
gap> runtime;
667
gap> runtime_quiver;
1578
```

Reply:
Sorry, my first numbers were wrong, now they are fixed.

Reply:
You see from the timings that with PR https://github.com/homalg-project/HigherHomologicalAlgebra/pull/158 the runtime of `QuiverRows` is dropping.